src/quatrex/electron/__init__.py

```diff
@@ -1,8 +1,14 @@
 # Copyright 2023-2024 ETH Zurich and the QuaTrEx authors. All rights reserved.
 
 from quatrex.electron.solver import ElectronSolver
-from quatrex.electron.sse_coulomb_screening import SigmaCoulombScreening
+from quatrex.electron.sse_coulomb_screening import SigmaCoulombScreening, SigmaFock
```

Comment:
Why not `SigmaHartree` & `SigmaFock`?

Reply:
This is a bit arbitrary. I only included the Fock part as that is done in the old code. However, this is incorrect since the Wannier Hamiltonian contains both the Hartree and Fock part of the ground state system. Ideally, those parts should be removed from the Wannier Hamiltonian and for each iteration you calculate the Hartree and Fock parts.

Reply:
100% agreed that we should be removing XC stuff from the DFT Hamiltonian when we start putting XC on top with this GW transport code (possible student project?).

But i still don't fully understand why you call it `SigmaCoulombScreening` here. Is that not the Hartree part?

Reply:
Ah, SigmaCoulombScreening is the GW part